cmd/example-cloudinit-hook-sidecar/cloudinit.go

```diff
@@ -100,7 +101,7 @@ func (s v1alpha2Server) PreCloudInitIso(ctx context.Context, params *hooksV1alph
 func main() {
 	log.InitializeLogging("cloudinit-hook-sidecar")
 
-	socketPath := hooks.HookSocketsSharedDirectory + "/cloudinit.sock"
+	socketPath := filepath.Join(hooks.HookSocketsSharedDirectory, "/cloudinit.sock")
```

Comment:
The `/` before the filename should be removed.

Reply:
Done